### PR TITLE
5️⃣ refactor(services: stripe billing): migrate from fastify to @whatwg-node/server

### PR DESCRIPTION
**Pull request by YassinEldeeb**
related to https://github.com/kamilkisiela/graphql-hive/issues/657 but doesn't close it entirely

### CODE REVIEW

File: packages/services/stripe-billing/src/router.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  stripe: stripeApi,
+  stripeData$: loadStripeData$,
+};
+
+const respondWithTRPC = createFetchAPIHandler({
+  router: stripeBillingApiRouter,
+  createContext: async (): Promise<Context> => context,
+});
+
+stripeBillingRouter.all('/trpc/:path+', async (req: any) => {
```
Comment (ardatan) [inline packages/services/stripe-billing/src/router.ts:46]:
```suggestion
stripeBillingRouter.all('/trpc/:path+',  createFetchAPIHandler({
  router: stripeBillingApiRouter,
  createContext: async (): Promise<Context> => context,
}))
```

Comment (ardatan) [reply packages/services/stripe-billing/src/router.ts:46]: We can simplify the code like that

Comment (YassinEldeeb) [reply packages/services/stripe-billing/src/router.ts:46]: Oh nice! I didn't know you could do that. I just followed the example there: https://github.com/esamattis/trpc-cloudflare-worker/blob/master/src/worker.ts#L3

**Comment by YassinEldeeb** [comment]
5️⃣

**Comment by dotansimha** [comment]
Closing. This PR is outdated and we'll reconsider it later.

**Pull request closed without merging**